packages/bpk-mixins/_typography.scss

```
@@ -774,6 +774,77 @@
   margin-bottom: bpk-spacing-sm();
 }
 
+/// Underlined part inside an inline link.
+///
+/// @example scss
+///   .selector {
+///     @include bpk-link-underlined();
+///   }
+
+@mixin bpk-link-underlined {
+  padding-bottom: 0.03em;
```

Should it be calc(-0.22em)?
```suggestion
  padding-bottom: (-0.22em);
```